app/views/users/_user.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<div class="sections">
+    <%= image_tag (user.photo), height: '115', width: '115' %>
+    <div class="users">
+        <h1><%= link_to (user.name), user%></h1>
+        <span>Number of posts: <%=user.posts_counter%></span>
```

Comment:
User index views can not find `#posts_counter` which gives an error. Kindly fix the error.
![posts_counter](https://user-images.githubusercontent.com/35504149/172709041-45d52ee6-8009-461c-9498-25248defd856.png)